webapp/components/admin_console/connection_settings.jsx

```diff
@@ -51,7 +51,7 @@ export default class ConnectionSettings extends AdminSettings {
                     label={
                         <FormattedMessage
                             id='admin.service.corsTitle'
-                            defaultMessage='Allow Cross-origin Requests from:'
+                            defaultMessage='Enable Cross-Origin Requests:'
```

Comment:
Propose keeping "from" so it's clear we're defining a value, rather than having a true/false setting.